CHANGELOG.md

```diff
@@ -5,6 +5,159 @@ The format is a variant of [Keep a Changelog](https://keepachangelog.com/en/1.0.
 
 This project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html). Breaking changes should trigger an increment to the major version. Features increment the minor version and fixes or other changes increment the patch number.
 
+<a name="v0.9.0"></a>
+## [v0.9.0] - 2022-04-21
+
+### Feat
+
+- Index Command (#871)
+
+   - Lily received a new command - **index** - The index command can be used to index a single tipset, referenced by either its height or its TipSetKey. This command will block until indexing the tipset has completed.
```

Comment:
Is `TipSetKey` the CID of the tipset?

Reply:
Yeah basically; Its the CID's of the blocks in the tipset: https://github.com/filecoin-project/lotus/blob/d794b49df352ce9f909baa5abb520848c1aa26b6/chain/types/tipset_key.go#L41
Example:
`{bafy2bzacebc2o5p5t5ie6ovr7mmploapc5wzjqqghqdxsvi725efdat36aj7y,bafy2bzacecrth4v72gbv7nyk46znorge6kqnh5ylj5mhpbe46fvguq7lzp7zm,bafy2bzacedk6iasod3w5cttpuaypczooqfl4ocolry5uap25nmzxsxcogu6qy,bafy2bzacedsui2bola6ovy7y3p5xmgeui5e2ak7nsfwafg37m2eogpv7wdxbw,bafy2bzacedwt6sz5jbwzo47vun77ylbmixuuai7vda6zzgjsgnzpfu6vbrvoi,bafy2bzacecq2h4boyk35el7sn4c32v2nzk5hpt4yqqeuucmzxohzprievolzu}`